src/Panel.jsx

```diff
@@ -35,12 +49,14 @@ class CollapsePanel extends Component {
       [`${prefixCls}-item-disabled`]: disabled,
     }, className);
     return (
-      <div className={itemCls} style={style} id={id} role="tablist">
```

Comment:
Why remove `role="tablist"`?

Reply:
I just moved it down to an inner div, since what we want to focus would be the header instead of the whole panel.